jans-client-api/server/src/main/resources/swagger.yaml

```diff
@@ -1,2886 +0,0 @@
-# raw swagger spec link:
```

Comment:
Where is new swagger file located? I somehow can't find it. We should be able to call jans-client-api with swagger and be able see updated docs.